android/build.gradle

```diff
@@ -737,7 +737,7 @@ afterEvaluate {
 
 if (CLIENT_SIDE_BUILD) {
     def aarDir = "${buildDir}/outputs"
-    def aar = file("${aarDir}/android-debug.aar")
+    def aar = file("${aarDir}/android-${resolveBuildType()}.aar") // e.g. android-debug.aar
```

Comment:
I'm not sure what these line actually do. I suspect it's something related to building Android part of npm package. Do you think we can leave `android-debug.aar` here? Do we even have two different builds for debug and release mode?